cmd/infra/powervs/create.go

```diff
@@ -1001,12 +1001,9 @@ func (infra *Infra) setupPowerVSDhcp(options *CreateInfraOptions, session *ibmpi
 	// only one dhcp server is allowed per cloud instance
 	// if already a dhcp server existing in cloud instance use that instead of creating a new one
 	if len(dhcpServers) > 0 {
```

Comment:
nit: The block of code going from here could be a unit tested function which receives dhcpServers as a parameter

Reply:
sure thanks for the input, moved the code block to a separate func.

Reply:
Added the UT for this func. Ptal.